src/test/java/com/crud/tasks/trello/mapper/TrelloMapperTest.java

```diff
@@ -1,9 +1,130 @@
 package com.crud.tasks.trello.mapper;
 
+import com.crud.tasks.domain.*;
+import com.crud.tasks.mapper.TrelloMapper;
+import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class TrelloMapperTest {
 
+    @Test
+    public void mapToBoardTest(){
+
+        //Given
+        TrelloList trelloList1 = new TrelloList("id_list1", "name_list", true);
+        TrelloList trelloList2 = new TrelloList("id_list2", "name_list", true);
+        List<TrelloList> trelloListsList = new ArrayList<>();
+        trelloListsList.add(trelloList1);
+        trelloListsList.add(trelloList2);
+
+        TrelloBoardDto trelloBoardDto1 = new TrelloBoardDto();
+        TrelloBoardDto trelloBoardDto2 = new TrelloBoardDto();
+        List<TrelloBoardDto> trelloBoardDtosList = new ArrayList<>();
+        trelloBoardDtosList.add(trelloBoardDto1);
+        trelloBoardDtosList.add(trelloBoardDto2);
+
+        //When
+        TrelloMapper trelloMapper = new TrelloMapper();
```

Comment:
trelloMapper jest wykorzystywane we wszystkich testach. Powinien byc zmienna klasowa